docs/source/user/index.md

```diff
@@ -1,6 +1,6 @@
 # User Guide
 
-In-depth guides, tutorials, and how-tos for using JupyterLab.
+Use this page to navigate to in-depth guides, tutorials, and how-tos for using `JupyterLab <../index>`_.
```

Comment:
This link does not work:

![Screenshot from 2023-05-24 18-26-04](https://github.com/jupyterlab/jupyterlab/assets/5832902/018ff2cf-2bdd-4f01-9100-4a86a33bdc58)


As also seen in failing check-links job:

```
_ /home/runner/work/jupyterlab/jupyterlab/docs/source/getting_started/overview.rst: ../index _
../index: No such file: /home/runner/work/jupyterlab/jupyterlab/docs/source/getting_started/../index
```

Reply:
Done